### PR TITLE
Added initial health endpoint

### PR DESCRIPTION
**Pull request by tfrank64**

### CODE REVIEW

**Comment by tfrank64** [comment]
Assigning the review to @tunniclm as we have already spoke about this.

**Comment by seabaylea** [comment]
The `manifest.yml` file also needs to be updated to include a reference to the HTTP endpoint, e.g..

```
health-check-type: http
health-check-http-endpoint: /health
```

We also need something added to the Kubernetes config file (once we have one!).

File: refresh/templates/common/Application.swift [hunk 1]
```diff
@@ -67,6 +67,10 @@ public func initialize() throws {
 <% if (exampleEndpoints) { -%>
     initializeProductRoutes()
 <% } -%>
+
+    router.get("/health") { request, response, _ in
+        try response.send("{\"status\":\"UP\"}").end()
```
Comment (tunniclm) [inline refresh/templates/common/Application.swift]: We should probably be setting the Content-Type of the response to indicate the JSON data content.

Comment (tunniclm) [reply refresh/templates/common/Application.swift]: If you use `send(json:)` it will do this for you.

**Comment by tfrank64** [comment]
@seabaylea When adding the health check lines to my manifest.yml, I get the following error when pushing `Server error, status code: 400, error code: 100001, message: The app is invalid: health_check_type must be one of port, none, process`. `http` seems to be a valid type in the docs, but I still get this error, any ideas?

**Comment by seabaylea** [comment]
@tfrank64 Based on this question on StackOverflow:
https://stackoverflow.com/questions/39736774/health-check-in-cloud-foundry

It may be that its not yet available for the version of CloudFoundry that we have deployed in Bluemix...

**Comment by tfrank64** [comment]
@seabaylea Yes, looks like that is possible. Not sure of a way to check the cf version on Bluemix?

Additionally, it still doesn't work if I deploy the app to Bluemix and then run:
 `cf set-health-check myApp http --endpoint /health`

**Review by tunniclm — APPROVED** [review]

**Comment by tunniclm** [comment]
The changes look good. Let me know when you have resolved the Bluemix health-check stuff and are ready to merge.

**Comment by rolivieri** [comment]
According to CF docs (https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html#types), `http` should be a valid health check method... but when running this command against Bluemix, it complains about `http` not being a valid method:

```
Ricardos-MacBook-Pro:swift-helloworld olivieri$ bx cf push -u http -t 10
Invoking 'cf push -u http -t 10'...

Using manifest file /Users/olivieri/git/swift-helloworld/manifest.yml

FAILED
Error: Invalid health-check-type param: http
```

To move forward, I will open a separate work item that is about the required changes for CF and Kubernetes in regards to health checking. This work item was just about adding the health endpoint to the generated code.

Also, we will open a ticket against Bluemix to ask about why `http` is not supported yet as a health check method.

**Comment by tunniclm** [comment]
Will merge when tests complete.

**Pull request merged**